tests/noise/test_noisy_execution.py

```diff
@@ -12,10 +14,27 @@
     ExpectationMeasure,
     Observable,
 )
-from mpqp.execution import ATOSDevice, AvailableDevice, AWSDevice, IBMDevice, run
+from mpqp.execution import (
+    ATOSDevice,
+    AvailableDevice,
+    AWSDevice,
+    GOOGLEDevice,
+    IBMDevice,
+    run,
+)
 from mpqp.gates import *
 from mpqp.noise import AmplitudeDamping, BitFlip, Depolarizing, PhaseDamping
 from mpqp.tools.errors import UnsupportedBraketFeaturesWarning
+from mpqp.tools.theoretical_simulation import validate_noisy_circuit
+
+noisy_devices: list[Any] = [
+    dev
+    for dev in list(ATOSDevice) + list(AWSDevice) + list(IBMDevice) + list(GOOGLEDevice)
+    if dev.is_noisy_simulator()
+]
+# TODO: in the end this should be automatic as drafted above, but for now only
+# one device is stable
+noisy_devices = [AWSDevice.BRAKET_LOCAL_SIMULATOR]
```

Comment:
We can add IBMDevice.AER_SIMULATOR now for instance

Reply:
done